examples/Boost-python/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2013, Ruslan Baratov
+# Modified work: Copyright (c) 2018, Gregory Kramida
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+option(HUNTER_BUILD_SHARED_LIBS "..." ON)
+
+# Configure:
+set(REQUESTED_PYTHON_VERSION 3.5)
+set(TESTING_CONFIG_OPT FILEPATH "${CMAKE_CURRENT_LIST_DIR}/config.cmake")
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+# Requires python version 3.6.7. Change this in config.cmake if necessary.
```

Comment:
> Perhaps change or remove comment on line 18 to match this.

Yes, missed that. Tests already running, let's do it as a follow-up.

Reply:
Done: https://github.com/ruslo/hunter/commit/c035cad1523d4270864455dae4c95344e47ed60a